docs/api/components/schemas.yaml

```
@@ -963,6 +963,29 @@ Replication-status:
   required:
     - replication_id
   title: Replication-status
+Scopes:
+  description: A map of all the collections with their corrosponding configs for this scope
+  type: object
+  properties:
+    collection_name:
+      $ref: '#/CollectionConfig'
```

`collection_name` isn't a literal, should be a user-definable property that has a CollectionsConfig type

```suggestion
    collections:
      description: The set of collections on the scope
      type: object
      additionalProperties:
        $ref: '#/CollectionConfig'
```

---

Makes sense. Forgot about that property. Saw a couple other places which didn't use this when they should have so will change them as well.